vitrina/datasets/templates/vitrina/datasets/detail.html

```diff
@@ -114,28 +99,28 @@ <h3 class="custom-title is-size-4-mobile no-margin-bottom">{{ object.title }}</h
                 <p class="dataset-meta-title m-b-md">{% translate "Dalintis socialiniuose tinkluose" %}</p>
                 <p class="social-icons m-t-md">
                     <a target="_blank"
-                       href="https://www.facebook.com/sharer/sharer.php?u=https://data.gov.lt/dataset/{{ dataset.slug }}"
+                       href="https://www.facebook.com/sharer/sharer.php?u=https://data.gov.lt{{ dataset.get_absolute_url }}"
                        class="dataset-meta-link">
                         <span class="icon is-gray">
                             <i class="fab fa-facebook-square fa-2x"></i>
                         </span>
                     </a>
                     <a target="_blank"
-                       href="https://www.linkedin.com/sharing/share-offsite/?url=https://data.gov.lt/dataset/{{ dataset.slug }}&title={{ dataset.title }}"
+                       href="https://www.linkedin.com/sharing/share-offsite/?title={{ dataset.title }}&url=https://data.gov.lt{{ dataset.get_absolute_url }}"
                        class="dataset-meta-link m-l-sm">
                         <span class="icon is-gray">
                             <i class="fab fa-linkedin fa-2x"></i>
                         </span>
                     </a>
                     <a target="_blank"
-                       href="http://twitter.com/share?text={{ dataset.title }}&url=https://data.gov.lt/dataset/{{ dataset.slug }}"
+                       href="http://twitter.com/share?text={{ dataset.title }}&url=https://data.gov.lt{{ dataset.get_absolute_url }}"
```

Comment:
`data.gov.lt` should not be hardcoded, because we will have multiple instances, at least `data.gov.lt` and `staging.data.gov.lt`. Please fix this, otherwise, it will be forgotten.

See: https://docs.djangoproject.com/en/3.2/ref/contrib/sites/#getting-the-current-domain-for-full-urls

Reply:
Please search the code and fix all the places, where data.gov.lt is hardcoded.

Reply:
One possible solution, how this can be easily fixed, is by adding current domain into a [context processor](https://docs.djangoproject.com/en/3.2/ref/templates/api/#writing-your-own-context-processors) or creating a [template tag](https://docs.djangoproject.com/en/3.2/howto/custom-template-tags/#howto-writing-custom-template-tags).

Also, keep in mind, that URI protocol part should not be hardcoded too, see:

- https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.build_absolute_uri
- https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.is_secure